docs/modules/ROOT/pages/mathematical-operators.adoc

```diff
@@ -0,0 +1,93 @@
+[[mathematical-operators]]
+= Mathematical operators
+
+Mathematical operators are a handy way to update numerical fields in a single roundtrip to the DB.
+
+Specific operators are available on different numerical types (`Int`, `Float`, xref::type-definitions/types.adoc#type-definitions-types-bigint[`BigInt`]).
+
+We chose to keep operators separated by types as some operators are not available to Natural numbers.
```

Comment:
This feels redundant, given the statement above

```suggestion
```

Reply:
Agree!